materialize-bigquery/testdata/spec.json

```diff
@@ -0,0 +1 @@
}},"field_selection":{"keys":["key1","key2","boolean","integer","string"],"values":["defAULT","number","person@place!","source.name","with-dash"],"document":"flow_document"},"journal_read_suffix":"materialize/test/sqlite/key_value"},{"resource_config_json":{"table":"no_values"},"resource_path":["no_values"],"collection":{"name":"no_values","write_schema_json":{"$id":"file:///builder/materialize-bigquery/testdata/flow.yaml?ptr=/collections/no_values/schema","properties":{"key1":{"type":"integer"},"key2":{"type":"boolean"}},"required":["key1","key2"],"type":"object"},"key":["/key1","/key2"],"uuid_ptr":"/_meta/uuid","projections":[{"field":"flow_document","inference":{"types":["object"],"exists":1}},{"ptr":"/key1","field":"key1","is_primary_key":true,"inference":{"types":["integer"],"exists":1}},{"ptr":"/key2","field":"key2","is_primary_key":true,"inference":{"types":["boolean"],"exists":1}}],"ack_template_json":{"_meta":{"ack":true,"uuid":"DocUUIDPlaceholder-329Bb50aa48EAa9ef"}},"partition_template":{"name":"no_values","replication":3,"labels":{"labels":[{"name":"app.gazette.dev/managed-by","value":"estuary.dev/flow"},{"name":"content-type","value":"application/x-ndjson"},{"name":"estuary.dev/build","value":"catalog"},{"name":"estuary.dev/collection","value":"no_values"}]},"fragment":{"length":536870912,"compression_codec":2,"stores":["s3://a-bucket/"],"refresh_interval":300000000000,"retention":0,"flush_interval":0,"path_postfix_template":"utc_date={{.Spool.FirstAppendTime.Format \"2006-01-02\"}}/utc_hour={{.Spool.FirstAppendTime.Format \"15\"}}"},"flags":4,"max_append_rate":4194304}},"partition_selector":{"include":{"labels":[{"name":"estuary.dev/collection","value":"no_values"}]},"exclude":{"labels":null}},"field_selection":{"keys":["key1","key2"],"document":"flow_document"},"journal_read_suffix":"materialize/test/sqlite/no_values"}],"shard_template":{"id":"materialize/test/sqlite","sources":null,"recovery_log_prefix":"recovery","hint_prefix":"/estuary/flow/hints","hint_backups":2,"max_txn_duration":1000000000,"min_txn_duration":0,"labels":{"labels":[{"name":"app.gazette.dev/managed-by","value":"estuary.dev/flow"},{"name":"estuary.dev/build","value":"catalog"},{"name":"estuary.dev/log-level","value":"info"},{"name":"estuary.dev/task-name","value":"test/sqlite"},{"name":"estuary.dev/task-type","value":"materialization"}]},"ring_buffer_size":65536,"read_channel_size":131072},"recovery_log_template":{"name":"recovery/materialize/test/sqlite","replication":3,"labels":{"labels":[{"name":"app.gazette.dev/managed-by","value":"estuary.dev/flow"},{"name":"content-type","value":"application/x-gazette-recoverylog"},{"name":"estuary.dev/build","value":"catalog"},{"name":"estuary.dev/task-name","value":"test/sqlite"},{"name":"estuary.dev/task-type","value":"materialization"}]},"fragment":{"length":268435456,"compression_codec":4,"stores":["s3://a-bucket/"],"refresh_interval":300000000000,"retention":0,"flush_interval":0},"flags":4,"max_append_rate":4194304}}
```

Comment:
These will be hard to change if we need to modify or extend the materialization specs for the tests. Ideally I think we'd have some tool that would generate a spec from a `flow.yaml` file, but I don't think that's necessary now. 

For the time being, can we store these as formatted JSON so they are at least somewhat human-readable if we need to make adjustments?